Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs

```diff
@@ -2250,4 +2250,22 @@ public enum FadeSpeedChoice
     }
 
     #endregion
+
+    #region Enum KryptonMessageBoxResult
+
+    public enum KryptonMessageBoxResult : int
+    {
+        None = DialogResult.None,
+        Ok = DialogResult.OK,
+        Cancel = DialogResult.Cancel,
+        Abort = DialogResult.Abort,
+        Retry = DialogResult.Retry,
+        Ignore = DialogResult.Ignore,
+        Yes = DialogResult.Yes,
+        No = DialogResult.No,
+        Checked = 8,
```

Comment:
Move these to higher numbers,
And Changed CheckState to be Intermediate>
Currently only 1 of these states can be returned

Reply:
@Smurf-IV Any numbers to avoid, i.e. can they be 31 & 32?

Reply:
It's an integer, so go big ;-)

Reply:
@Smurf-IV Should it be a `uint` as all default values come from [here](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-messagebox)?

Reply:
No, As they are Return values, and as such should be `int` according to that link.
But consider this scenario..
- Question is `MB_CANCELTRYCONTINUE` based
- Checkbox is set as an option
- How does it return a Continue with a checkbox intermediate state set ?

In other words, I do not think returning the checkstate is going to work.
And only the `get` CheckState is required as an API !

Reply:
There might be another way, just use `Checked` as it's a bool. Checked would be yes & unchecked would be no?

Reply:
Used higher numbers (1006 & 1007)